pageserver/src/metrics.rs

```diff
@@ -495,7 +565,9 @@ impl Drop for TimelineMetrics {
 }
 
 pub fn remove_tenant_metrics(tenant_id: &TenantId) {
-    let _ = STORAGE_TIME.remove_label_values(&["gc", &tenant_id.to_string(), "-"]);
+    let _ = STORAGE_TIME_SUM_PER_TIMELINE.remove_label_values(&["gc", &tenant_id.to_string(), "-"]);
+    let _ =
+        STORAGE_TIME_COUNT_PER_TIMELINE.remove_label_values(&["gc", &tenant_id.to_string(), "-"]);
```

Comment:
```suggestion
    let tid = tenant_id.to_string();
    let _ = STORAGE_TIME_SUM_PER_TIMELINE.remove_label_values(&["gc", &tid, "-"]);
    let _ =
        STORAGE_TIME_COUNT_PER_TIMELINE.remove_label_values(&["gc", &tid, "-"]);
```

Reply:
Since the name now changed, why include the timeline_id="-" at all? Does it give some metric storage backend specific benefit?

Reply:
Inclined to just remove these.

Reply:
Aah nope, oki so we still have some metrics which are per timeline, but not just gc.

Reply:
Nope, this had me misunderstanding this so many times, removed both of these lines because there will never be such values because timeline_id is never `-`. It may have previously been.